about.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/a36c13d2c3.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./Style/about.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <nav class="top-nav">
+      <ul>
+        <li>
+          <a href="#"> <i class="fa fa-facebook"></i> </a>
+        </li>
+        <li>
+          <a href="#"> <i class="fa fa-twitter"></i> </a>
+        </li>
+        <li><a href="#"> English </a></li>
+        <li><a href="#"> My Page </a></li>
+        <li><a href="#" class="lastA"> Logout </a></li>
+      </ul>
+    </nav>
+
+    <nav class="web-nav">
+      <a href="index.html" class="brand"
+        ><img src="./images/logo.png" alt="brand-img"
+      /></a>
+      <ul>
+        <li><a href="about.html"> About </a></li>
+        <li><a href="#"> Program </a></li>
+        <li><a href="#"> Join </a></li>
+        <li><a href="#"> Sponsor </a></li>
+        <li><a href="#"> News </a></li>
+        <li><a href="#" class="lastCCA"> CC Campaign </a></li>
+      </ul>
+    </nav>
+
+    <nav class="mobile-nav">
+      <ul>
+        <li class="close_btn">
+          <i class="fa-solid fa-xmark" onclick="hamburger()"></i>
+        </li>
+        <li><a href="about.html"> About </a></li>
+        <li><a href="#"> Program </a></li>
+        <li><a href="#"> Join </a></li>
+        <li><a href="#"> Sponsor </a></li>
+        <li><a href="#"> News </a></li>
+        <li><a href="#"> CC Campaign </a></li>
+      </ul>
+    </nav>
+    <header class="Landing_Page">
```

Comment:
@saifullah767 even though you should make it easier for the user to go to the home page, Microverse requires you to put a link to the home page in the logo because most the websites do that but that doesn't mean you shouldn't add  a link to the home page in the navbar